### ISSUE TITLE
Downloaded file won't play

### ISSUE THREAD

**Issue by infinit-X** [Matrix86/flowdownloader#3]
After It downloaded the file, it is still unplayable with VLC. It detected the key and downloaded and decyrpted the file. But doesn;t work still.

**Comment by Matrix86** [comment]
I need more info to debug it. Can you share more info with me? Also privately is ok

**Comment by infinit-X** [comment]
can i have your email pls ?

**Comment by infinit-X** [comment]
How I can contact you ? I'm not subscribed to X premium so i can't contact you there

I will share some info here, So below is a link generated by flowdownloader chrome extension

"flowdownloader -k "t9AEm7ccwDuTS0uPjn4BFg==" -u https://ecvod.orbitguard.com/vod/5cimxu7uwnd.mp4/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9Mi8xOS8yMDI0IDEyOjAyOjE0IFBNJmhhc2hfdmFsdWU9a0M2MmRUSHFzd0tHRG4zdnp5cU0xUT09JnZhbGlkbWludXRlcz0x"

And this link is valid only for 1 minute cuz otherwise it gives https 403 forbidden error. So when i ran this on flowdownloader it download all the .ts parts and decrypt them and output .mp4 file. But when I try to play it, it doesn't play just like I only downloaded the parts without decrypting. I tried the same procedure on this sample website 

"https://v.ggicci.me/sample/"

there everything works fine and the downloaded video is playable
